src/frontend/komponenter/Fagsak/Saksoversikt/utils.tsx

```diff
@@ -226,25 +234,30 @@ export const lagLenkePåResultat = (
                 </Link>
             );
         case Saksoversiktbehandlingstype.KLAGE: {
-            const LenkeTilKlage = () => (
-                <Link
-                    href={`/redirect/familie-klage/behandling/${behandling.id}`}
-                    onMouseDown={e => e.preventDefault()}
-                    target="_blank"
-                >
-                    <span>{utledKlageBehandlingsresultatTilTekst(behandling)}</span>
-                    <ExternalLinkIcon fontSize={'1.5rem'} />
-                </Link>
-            );
-            return ankeHarEksistertPåBehandling(behandling) ? (
-                <Tooltip content="Det finnes informasjon om anke på denne klagen. Gå inn på klagebehandlingens resultatside for å se detaljer.">
-                    <ResultatCelle>
-                        <StatusIkon status={Status.ADVARSEL} />
-                        <LenkeTilKlage />
-                    </ResultatCelle>
-                </Tooltip>
-            ) : (
-                <LenkeTilKlage />
+            return (
+                <ResultatCelle>
+                    <Link
+                        href={`/redirect/familie-klage/behandling/${behandling.id}`}
+                        onMouseDown={e => e.preventDefault()}
+                        target="_blank"
```

Comment:
Mest sannsynlig et non-issue men kan vurderes, se her https://perishablepress.com/fix-blank-target-vulnerability/

Reply:
Kjenner til det, men tenker det ikke er et problem her som vi lenker mellom to interne systemer vi selv kontrollerer 😇